shared/src/test/scala/squants/time/FrequencySpec.scala

```diff
@@ -62,6 +69,26 @@ class FrequencySpec extends FlatSpec with Matchers {
     Hertz(1) * Seconds(1) should be(Each(1))
   }
 
+  it should "return the TimeDerivative when multiplied by a TimeIntegral" in {
+    Hertz(100) * FeetPerSecondSquared(100) should be(FeetPerSecondCubed(10000))
+    Hertz(100) * Radians(100) should be(RadiansPerSecond(10000))
+    Hertz(100) * Each(100) should be(Hertz(10000))
+    Hertz(100) * Coulombs(100) should be(Amperes(10000))
+    Hertz(100) * WattHours(100) should be(Watts(10000))
+    Hertz(100) * Newtons(100) should be(NewtonsPerSecond(10000))
+    Hertz(100) * Bytes(100) should be(BytesPerSecond(10000))
+    Hertz(100) * Meters(100) should be(MetersPerSecond(10000))
+    Hertz(100) * LumenSeconds(100) should be(Lumens(10000))
+    Hertz(100) * LuxSeconds(100) should be(Lux(10000))
+    Hertz(100) * Webers(100) should be(Volts(10000))
+    Hertz(100) * Kilograms(100) should be(KilogramsPerSecond(10000))
+    Hertz(100) * NewtonSeconds(100) should be(Newtons(10000))
+    Hertz(100) * Watts(100) should be(WattsPerHour(10000))
```

Comment:
Good catch.  Most time derivative relationships are based on the second.  Energy/Power/PowerRamp prefer the WattHour, Watt and WattsPerHour units, so the base time is one hour.  The new conversion assumed 1 second, which coupled with my blind paste and copy test setup, led to a passing test on a bad conversion.  It's fixed up now.

Thanks!